README.md

```diff
@@ -147,6 +147,10 @@ filter = ["id", "name"]
 
 In the above example, we will only sync MySQL table tfiler's columns `id` and `name` to Elasticsearch. 
 
+## Keep in mind
+
++ If you try to sync the same data twice the program will skip the `mysqldump` proccess and only listen to changes in the bin log. To force `mysqldump` again you must delete `./var/master.info`
```

Comment:
Works for me when deleting the `~/go/src/github.com/siddontang/go-mysql-elasticsearch/var/master.info`. I think it would help people know.